components/director/internal/domain/formationassignment/notifications.go

```diff
@@ -92,6 +92,21 @@ func (fan *formationAssignmentNotificationService) GenerateFormationAssignmentNo
 }
 
 func (fan *formationAssignmentNotificationService) PrepareDetailsForNotificationStatusReturned(ctx context.Context, tenantID string, fa *model.FormationAssignment, operation model.FormationOperation) (*formationconstraint.NotificationStatusReturnedOperationDetails, error) {
+	var targetType model.ResourceType
+	switch fa.TargetType {
+	case model.FormationAssignmentTypeApplication:
+		targetType = model.ApplicationResourceType
+	case model.FormationAssignmentTypeRuntime:
+		targetType = model.RuntimeResourceType
+	case model.FormationAssignmentTypeRuntimeContext:
+		targetType = model.RuntimeContextResourceType
+	}
```

Comment:
maybe it's a good idea to have a default case that returns error in case the fa.TargetType don't match any of the mentioned?

Reply:
It will fail in `getObjectSubtype` with `errors.Errorf("unknown formation type %s", objectType)`